README.md

```diff
@@ -32,6 +32,16 @@ NB: this assumes you are running on the GOV.UK virtual machine, not your host.
   ./install # git fetch from each dependency dir and bundle install
 ```
 
+Run using bowler:
+```
+bowl smartanswers
```

Comment:
You bowl from cd /var/govuk/development/ on the VM or at least I do.

Happy to update this to reflect that